data/json/mapgen/garden_botanical.json

```diff
@@ -9,8 +9,8 @@
         "4": "f_null",
         "7": "f_null",
         "B": "f_bench",
-        "D": "f_dandelion",
-        "F": "f_flower_marloss",
```

Comment:
Removed marloss flowers, moved a couple plants and added more plants in greenhouse. This is the fix for the issue mentioned in the summary of this PR.